EventListener/ViewResponseListener.php

```diff
@@ -27,16 +28,22 @@
  */
 class ViewResponseListener extends TemplateListener
```

Comment:
this should not extend the TemplateListener anymore but just run earlier (and so no need for the container)